build.sbt

```diff
@@ -1,30 +1,55 @@
-enablePlugins(ScalaJSPlugin)
+// enablePlugins(ScalaJSPlugin)
 
-enablePlugins(ScalaJSBundlerPlugin)
+// enablePlugins(ScalaJSBundlerPlugin)
 
 // @TODO[Security] Is this a good idea to leave this here long term?
 // resolvers += Resolver.sonatypeRepo("snapshots")
 
-libraryDependencies ++= Seq(
-  "com.raquo" %%% "airstream" % "0.10.0",
-  "com.raquo" %%% "domtypes" % "0.10.1",
-  "com.raquo" %%% "domtestutils" % "0.12.0" % Test,
-  "org.scalatest" %%% "scalatest" % "3.1.1" % Test
-)
 
-scalacOptions ++= Seq("-deprecation", "-feature", "-language:higherKinds", "-language:implicitConversions")
+// lazy val core = (project )
 
-version in installJsdom := "16.2.0"
+lazy val jsdocs = project
+  .settings(
+   libraryDependencies += "org.scala-js" %%% "scalajs-dom" % "1.1.0"
+  )
+  .enablePlugins(ScalaJSPlugin)
+  .dependsOn(laminar)
 
-useYarn := true
+lazy val docs = project       // new documentation project
+  .in(file("myproject-docs")) // important: it must not be docs/
```

Comment:
If it must not be `docs` you could rename the project `lazy val` to something else too.
Like `laminar-docs` or `laminarDocs`

Reply:
This name actually doesn't matter, that's where mdoc will put compiled documentation:

https://scalameta.org/mdoc/docs/installation.html#sbt

I'll rename the folder to `laminar-docs`, but it's only a temporary one anyways.